android/app/src/main/java/org/openbot/autopilot/AutopilotFragment.java

```diff
@@ -275,6 +275,8 @@ private void recreateNetwork(Model model, Network.Device device, int numThreads)
 
   @Override
   public synchronized void onResume() {
+    croppedBitmap = null;
+    tracker = null;
```

Comment:
Why are these set to null here?

Reply:
So, there was a bug into the application, which is being fixed here by reinitializing the variables.

Bug was if you turn on the auto mode into the autopilot, then move to settings by clicking on the usb/ble icon and come back to the fragment. 

it crashes.

Reply:
Great 👍. One 🐛 less!